benchmark/README.md

```diff
@@ -24,7 +31,7 @@ The first part of this file defines the search terms (the "needles") `Lorem`, `s
 The second part, after a blank line, defines the corpus to search in  (the "haystack").
 Currently, these files **must be encoded as UTF-16 Little-Endian without BOM**.
```

Comment:
Yes, since the Java, Rust and Python benchmarks still expect their inputs to be UTF-16. I'd vote to create and issue to update those after the release.